site/docs/introduction/aggregate-programming.mdx

```diff
@@ -0,0 +1,112 @@
+---
+title: Aggregate Programming and Field Calculus 
+---
+
+# The Challenge of Maintaining Distributed Systems 
+
+The **progressive increase** in the number of interconnected devices, leads to a **rise in the costs** associated with maintaining distributed systems. This creates **significant challenges** in implementing software services on a **global scale** using the traditional approach of individually programming each agent. This situation drives the search for solutions aimed at improving the **autonomy** of computing systems and **reducing their complexity**.
+
+## Aggregate Programming as a Relevant Approach
+
+In this context, **aggregate programming**[<sup>1</sup>](#bibliography) emerges as a **relevant macroprogramming approach**. It is based on the **functional composition** of **reusable collective behavior blocks**, with the goal of efficiently achieving **complex** and **resilient behaviors** in **dynamic networks**.
+
+### Key Properties of Aggregate Programming
+
+- **Self-organizing**: the system as a whole is capable of achieving globally coordinated behaviors through decentralized interactions among individual components.
+    - **Self-coordination**: components autonomously exchange information and adapt their behavior to maintain coherence, even in the absence of centralized control;
+    - **Self-stabilization**: the system naturally converges to a desired stable state despite transient disruptions or changes in the network.
+- **Resilience**: the system is inherently robust to node failures, dynamic topology changes, and unpredictable environmental conditions, ensuring continued functionality under adverse circumstances.
+- **Scalability**: the system supports scalability across a wide range of system sizes, maintaining performance and efficiency as the number of components increases.
+
+These properties are initially studied on **basic building blocks**, such as:
+- **Broadcasting**
+- **Distance estimation**
+- **Data aggregation**
+- **Leader election**
+- **Clustering**
+- **Event detection**
+- **Consensus**
+- **Pattern formation**
+- **Gradient formation**
+- **Neighborhood monitoring**
+
+These building blocks are then transferred and validated on more **complex systems** created through **functional composition**.
+
+Aggregate programming finds its **conceptual roots** in **Field Calculus** and presents itself as a minimal functional programming language designed for the specification and composition of collective behaviors.
+
```

Comment:
Missing some AC applications examples, where can I use AC? What can I achieve with that? (drone swarms, ecc) For Another example look at the first published application of Collektive related to vascular morphogenesis: https://ieeexplore.ieee.org/document/10771167